go/client/chat_ui.go

```diff
@@ -115,7 +115,7 @@ func (c *ChatUI) ChatConfirmChannelDelete(ctx context.Context, arg chat1.ChatCon
 	if err != nil {
 		return false, err
 	}
-	return strings.TrimSpace(response) == confirm, nil
+	return response == confirm, nil
```

Comment:
a dev channel could contain a trailing space, this would prevent the prompt from ever being confirmed